acceptance_tests/README.md

```diff
@@ -0,0 +1,85 @@
+# Helm Acceptance Tests
+
+This directory contains the source for Helm acceptance tests.
+
+The tests are written using [Robot Framework](https://robotframework.org/).
+
+## System requirements
```

Comment:
is there a basic system requirement to run these tests? e.g. what are the specs of your PC running this test suite?